Libraries/dotNetRdf.Core/Writing/Formatting/NTriplesFormatter.cs

```diff
@@ -278,6 +279,17 @@ public override string FormatUri(string u)
         {
             return FormatChar(u.ToCharArray());
         }
+
+        /// <summary>
+        /// Matches the beginning of all lines.
+        /// </summary>
+        static readonly Regex commentLineRegex = new Regex(@"^", RegexOptions.Compiled | RegexOptions.Multiline);
+
+        /// <inheritdoc />
+        public virtual string FormatComment(string text)
+        {
+            return commentLineRegex.Replace(text, "#");
```

Comment:
Might be nicer to format the comment with a space after the `#` ?

Reply:
That's a good point, and as long as we are consistent in that approach in all of the formatters then I think it would be enough just to document it.